Quaver.Shared/Screens/Loading/MapLoadingScreen.cs

```diff
@@ -114,6 +111,20 @@ private void ParseAndLoadMap()
             }
         }
 
+        /// <summary>
+        /// Applies mods to the given map.
+        /// </summary>
+        /// <param name="qua"></param>
+        /// <param name="mods"></param>
+        public static void ApplyModsToQua(Qua qua, ModIdentifier mods)
```

Comment:
Reason for this being static?

Reply:
Doesn't use any properties. Do you think this would be better off as a method on `Qua`?